src/ansari/util/fastapi_helpers.py

```diff
@@ -0,0 +1,26 @@
+from fastapi import Depends, HTTPException, Request
+from jwt import PyJWTError
+
+from ansari.ansari_logger import get_logger
+from ansari.config import Settings, get_settings
+
+logger = get_logger()
+
+
+# Defined in a separate file to avoid circular imports between main_*.py files
+def validate_cors(request: Request, settings: Settings = Depends(get_settings)) -> bool:
```

Comment:
Like this! Are there othetr helpers that should be moved in here?

Reply:
Thanks!

Whenever I find code that is used in multiple files (e.g., `validate_cors` is used in main_api and main_whatsapp), I'll move them here, but now I can't think of any.